package.json

```diff
@@ -1,43 +1,46 @@
 {
   "name": "@spryker/oryx-for-zed",
   "version": "2.12.0",
-  "description": "Spryker ZED frontend automation tool (oryx based)",
+  "description": "Spryker ZED frontend automation tool",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:spryker/oryx-for-zed.git"
+  },
+  "license": "MIT",
   "author": {
     "name": "Spryker Sytems GmbH",
     "email": "info@spryker.com",
     "url": "http://spryker.com"
   },
-  "repository": {
-    "url": "git@github.com:spryker/oryx-for-zed.git",
-    "type": "git"
-  },
-  "license": "MIT",
   "main": "index.js",
   "engines": {
-    "node": ">=12.0.0",
-    "npm": ">=6.9.0"
+    "node": ">=16.0.0",
+    "npm": ">=8.0.0"
   },
   "dependencies": {
-    "@babel/core": "~7.17.12",
-    "@babel/preset-env": "~7.17.12",
-    "@spryker/oryx": "^1.4.0",
-    "autoprefixer": "~8.3.0",
-    "babel-loader": "~8.2.5",
-    "core-js": "~2.5.5",
+    "autoprefixer": "~9.8.8",
+    "babel-loader": "~8.3.0",
+    "colors": "~1.4.0",
+    "core-js": "~3.29.1",
```

Comment:
Please check if the `core-js` is still necessary.

Reply:
Removed